src/capitalsFirst.js

```diff
@@ -18,7 +18,20 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  let UpperStrArr = [];
+  let LowerStrArr = [];
+  let arrWords = str.split(' ');
+  for (let i = 0; i < arrWords.length; i++) {
+    if (arrWords[i][0].toLowerCase() === arrWords[i][0].toUpperCase()) {
+      continue;
+    } else if (arrWords[i][0].toLowerCase() === arrWords[i][0]) {
+      LowerStrArr.push(arrWords[i]);
+    } else {
+      UpperStrArr.push(arrWords[i]);
+    }
+  }
+  let concatTwoArr = UpperStrArr.concat(LowerStrArr);
+  str = concatTwoArr.join(' ');
+  return str;
```

Comment:
you can replace these 3 lines with 
return [...upperStrArr, lowerStrArr].join(' ')